test/python/WMCore_t/REST_t/Api_t.py

```diff
@@ -1,18 +1,27 @@
-from cherrypy.test import helper
-from cherrypy import expose, response, config as cpconfig
+# system modules
+import cjson, re, zlib
```

Comment:
I can change it to json, but I think it needs to be clarified by Seangchan.

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -1,18 +1,27 @@
> > -from cherrypy.test import helper
> > -from cherrypy import expose, response, config as cpconfig
> > +# system modules
> > +import cjson, re, zlib
> 
> I think json is every bit as good as cjson now (or actually is cjson)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6191/files#r38785305